fastlane/metadata/android/de/short_description.txt

```diff
@@ -0,0 +1 @@
+Client für den freien Media-Server
```

Comment:
```suggestion
Mobilclient für den freien Media-Server
```

To bring this in line with the English description. There might be better alternatives, but that's the best I could come up with.

Reply:
OK, let me top that.

Reply:
Thank you, that's better :sweat_smile:

Reply:
Now it must be nearly perfect :rofl: Of course you can adjust it anytime once it's merged. I was just trying to get you started with a proper base to build upon (and get rid of one minor show-stopper at that RFP over at F-Droid – so everything's ready once you have the G-free flavor). :crossed_fingers: